QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002147	$The Tombstone with a Stain
+QUEST_20150714_002148	$It seems that someone has already made a rubbing since there is a vague trace of ink. {nl}Shall we read it?
+QUEST_20150714_002149	$King Kadumel finally dies.
+QUEST_20150714_002150	$Instead of Mage Tower, the Tower of Fire was a more fitting name for Flurry's tower.
+QUEST_20150714_002151	$This is a very secluded area so monsters usually prey around here. {nl}Be careful around here.
+QUEST_20150714_002152	$The Research Records of Demetrius
+QUEST_20150714_002153	$In ancient times, humans lived with the Goddesses. {nl}When I first heard of this folklore, I thought it was made up in the respect for the Goddesses. {nl}I started to investigate what historic background caused such reverence as well as why this myth was created.
+QUEST_20150714_002154	$I increased the extent of my research. {nl}I travelled to many regions of the kingdom and collected similar cases. {nl}And I started to believe that this story was part of a bigger truth rather than just a small truth covered in fabrication.
+QUEST_20150714_002155	$The facts that we've discovered so far are thus: {nl}{nl}In the beginning, or a period not that far from the beginning, all humans used to live with the Goddesses underground. {nl}I suppose the Goddesses ruled over humans with more direct care. {nl}I have no idea why but soon, the humans left that place and scattered across the land.
+QUEST_20150714_002156	$This movement was made very naturally. {nl}If that's not so, there definitely must have been conflicts, and more fables would have been created.
+QUEST_20150714_002157	$I believe the Goddesses erased all memory and records of that time completely. {nl}However, we can't deny the possibility that another reason may exist due to the fact that this legend still remains even with the authority of the great Goddess.
+QUEST_20150714_002158	$And this is a very cautious prediction, but the creatures what we call devils may exist in the underground. {nl}Though that theory is hard to accept when we think about all the facts that are recorded in history and the present.
+QUEST_20150714_002159	$I can smell many Avietes. I heard they're effective… Could you share them with us?
+QUEST_20150714_002160	$Thank you. I will give you as many as you need. {nl}You will be successful.
+QUEST_20150714_002161	$I wanted to eat it…
+QUEST_20150714_002162	$You're not collecting Avietes?
+QUEST_20150714_002163	$I'll give them to you. {nl}You will be successful.
+QUEST_20150714_002164	$Our hope will definitely come true.
+QUEST_20150714_002165	$I trust Horaceus.
+QUEST_20150714_002166	$I'll cheer up and smile until the day comes.
+QUEST_20150714_002167	$I guess I'll take this tombstone rubbing to the person who wants it.
+QUEST_20150714_002168	$Since I've already made the rubbing, I'll give to the person who wants it.
```

Comment:
"I'll give to the person who wants it." -> "I'll give **it** to the person who wants it."